src/cpu-template-helper/src/utils/mock_kernel/Makefile

```diff
@@ -0,0 +1,16 @@
+ARCH := $(shell uname -m)
+NPROC := $(shell nproc)
+
+.PHONY: mock_kernel clean
+
+$(ARCH).bin: main.c
+ifeq ($(ARCH),x86_64)
+	gcc -nostdlib -fno-asynchronous-unwind-tables -s -o x86_64.bin main.c
+else
+	git clone --depth=1 git://git.kernel.org/pub/scm/linux/kernel/git/torvalds/linux.git
+	cd linux; make allnoconfig; make -j$(NPROC)
+	head -c 4096 linux/arch/arm64/boot/Image > aarch64.bin
```

Comment:
Yeah, I wasn't able to find an easy way to generate PE formatted mock kernel image, because it requires to have a special header in front of the actual PE image. I'll add that there.

Reply:
I managed to find a much easier way to generate a file acceptable. added it with some comments.